views/logout.go

```diff
@@ -0,0 +1,30 @@
+package views
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/ystv/web-auth/user"
+	"net/http"
+)
+
+// LogoutFunc Implements the logout functionality.
+// Will delete the session information from the cookie store
+func (v *Views) LogoutFunc(c echo.Context) error {
+	session, err := v.cookie.Get(c.Request(), v.conf.SessionCookieName)
+	if err != nil {
+		return v.errorHandle(c, err)
+	}
+
+	session.Values["user"] = user.User{}
+	session.Options.MaxAge = -1
+	err = session.Save(c.Request(), c.Response())
+	if err != nil {
+		return v.errorHandle(c, err)
+	}
+	// TODO Don't call env in this function have an initialiser
+	// then fetch from that store?
```

Comment:
Does this TODO get done?

Reply:
That was a Rhys TODO, I think I left it to understand what it meant, I should add a TODO: to understand the TODO
Or just remove it as I don't know what it means
Found where it was added - https://github.com/ystv/web-auth/commit/e904f62fb031600d26dcf1642521c62a53b43ec4 - 22/10/2020

Reply:
I don't believe it is needed